rest/tests/test_rest.py

```diff
@@ -255,8 +256,8 @@ def test_other_vocabulary(self):
 
     def test_term(self):
         """Test REST access for term"""
-        vocab_slug = self.create_vocabulary(self.repo.slug)['slug']
-        terms = self.get_terms(self.repo.slug, vocab_slug)
+        vocab1_slug = self.create_vocabulary(self.repo.slug)['slug']
```

Comment:
To make it clearer what I mean down below when it tests both vocabularies and both terms